core/src/main/java/io/confluent/kafka/schemaregistry/rest/resources/SubjectsResource.java

```diff
@@ -67,6 +70,8 @@ public void lookUpSchemaUnderSubject(final @Suspended AsyncResponse asyncRespons
                                        final @HeaderParam("Content-Type") String contentType,
                                        final @HeaderParam("Accept") String accept,
                                        @PathParam("subject") String subject,
+                                       @QueryParam("deleted") boolean
```

Comment:
There is a test case that specifically tests that  (RestAPITest.testDeleteSchemaVersionBasic line num #533). The default value for primitives is java default value. Thats the reason I didn't explicitly specify the annotation.